Iteration_4.rb

```diff
@@ -0,0 +1,66 @@
+require './lib/vehicle_factory'
+require './lib/dmv_data_service'
+require './lib/vehicle.rb'
+
+###### Option 3 ---> Analytics #######
```

Comment:
Remember, we want to practice OOP which means these methods could be encapsulated in a new class. Maybe a class called Analytics or something, then it should have a corresponding test file to test those methods. :)